Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -327,17 +323,21 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
     private func contentRows() -> [UIView] {
         var rows = [UIView]()
 
+        if hasMediaAttachment {
```

Comment:
<img width="520" alt="screen shot 2017-12-20 at 12 12 52 am" src="https://user-images.githubusercontent.com/217057/34193765-08ef580c-e51b-11e7-8361-ab5ffffe8227.png">

We now show the attachment+caption in the detail view. Design on this is TBD. I'm wondering if we can actually leverage the OWSMessageCell in this view to get the exact same layout in the detail screen as we do in the conversation view (though maybe special case oversized text messages), but it's not trivial. WDYT?

Reply:
Looks nice like that - I'd vote that we _not_ use the `OWSMessageCell` layout.  In particular, it'd be nice for media to _not_ be cropped in this view.